blocks/cards/cards.css

```diff
@@ -1,3 +1,8 @@
+.cards-card-body {
+  --content-spacing: var(--space-2);
+  --content-end-spacing: var(--space-5);
+}
+
 .cards {
   background-color: var(--calcite-ui-foreground-1);
```

Comment:
should really have `color` here. When ever you set background you should set foreground to ensure a11y. 

also keeps them grouped. You should be able to move the `color` from line 29 up here. Note: there is other work that fixes headers being hard coded to "white".

Reply:
Added color.